### PR TITLE
Fixes #1737

### PR DESCRIPTION
**Pull request by Jiashun97**
jasp-stats/INTERNAL-jasp#1714

### CODE REVIEW

**Comment by boutinb** [comment]
@Kucharssim Could you review it today? Then this could go to 0.16.1.

**Comment by Kucharssim** [comment]
I'll have a look

File: R/LSBuffonsneedlemanipulation.R [hunk 1]
```diff
@@ -95,8 +95,16 @@ LSBuffonsneedlemanipulation   <- function(jaspResults, dataset, options, state =
...
-        ggplot2::theme(legend.position = c(.17, .9))
+      if (xValue[which.max(propPost)] < 0.5){
+        propDistPlot$plotObject <-  propDistPlot$plotObject + 
+          ggplot2::theme(legend.position = c(.75, 1))
+      }
+      
+      if (xValue[which.max(propPost)] >= 0.5){
+        propDistPlot$plotObject <-  propDistPlot$plotObject + 
+          ggplot2::theme(legend.position = c(.25, 1))
+      }
```
Comment (Kucharssim) [inline R/LSBuffonsneedlemanipulation.R:106]:
```suggestion
      propDistPlot$plotObject <- propDistPlot$plotObject +
        ggplot2::theme(legend.position="right")
```

Move the legend entirely out of the plot area, which makes your life easier, and simplifies the code a lot.

Do this for every plot with a legend please.

**Review by Kucharssim — CHANGES_REQUESTED** [review]
This also relates to https://github.com/jasp-stats/INTERNAL-jasp/issues/1477

The legend situation is better, but not fixed completely, see
Screenshot 2022-02-08 at 13 48 11

That's why it's wise to put legend outside of the plotting area if you can, because otherwise you will fight with this issue forever.

Otherwise the code is ok, although you did not rebase properly: I wrote you on Slack with instructions what to do about it.

**Comment by boutinb** [comment]
@Jiashun97 Do you think you can make the change today? We would like to release tomorrow.

**Pull request closed without merging**